packages/analytics/analytics-chart/src/components/chart-types/TimeSeriesChart.vue

```diff
@@ -247,6 +252,19 @@ const tooltipDimensions = ({ width, height }) => {
   tooltipData.height = height
 }
 
+const handleChartClick = () => {
+  tooltipData.locked = !tooltipData.locked
+
+  if (chartInstance.value) {
+    if (tooltipData.locked) {
+      verticalLinePlugin.clickedSegment = chartInstance.value.chart.tooltip?.dataPoints[0]
+    } else {
+      verticalLinePlugin.clickedSegment = undefined
+    }
```

Comment:
can you simplify this into a ternary pls:

```
verticalLinePlugin.clickedSegment = tooltipData.locked
      ? chartInstance.value.chart.tooltip?.dataPoints[0]
      : undefined
```

Reply:
done